packages/front-end/components/Features/CodeSnippetModal.tsx

```diff
@@ -40,11 +40,6 @@ export function getApiBaseUrl(connection?: SDKConnectionInterface): string {
     );
   }
 
```

Comment:
ToDO is done once https://github.com/growthbook/terraform/pull/11/files is reviewed and deployed.